config/configgrpc/metadata.yaml

```diff
@@ -0,0 +1,108 @@
+config:
+  $id: https://opentelemetry.io/collector/configgrpc
```

Comment:
I'd like to see more ilke

```
config_file: ./my_component_schema.yaml
```

(also I expect some will have JSON-formatted JSON schema)